eth1/mock/client.go

```diff
@@ -0,0 +1,31 @@
+package mock
```

Comment:
Could be better to place it in `./eth1/test_utils.go` as this is the approach we have in other packages

Reply:
done